db/sql/src/channels.rs

```diff
@@ -486,6 +504,24 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_insert_corrupted_channel() -> anyhow::Result<()> {
+        let db = HoprDb::new_in_memory(ChainKeypair::random()).await?;
+
+        let a = Address::from(random_bytes());
+        let b = Address::from(random_bytes());
+
+        let channel_id = generate_channel_id(&a, &b);
+
+        db.insert_corrupted_channel(None, channel_id).await?;
+
+        db.get_corrupted_channel_by_id(None, &channel_id)
+            .await?
+            .context("corrupted channel must be present")?;
+
```

Comment:
Also add an assertion here that we cannot get that channel as a normal (uncorrupted) one.